tests/test_mc_simulator.py

```diff
@@ -29,14 +37,34 @@
     fixture_mantarray_mc_simulator_no_beacon,
 ]
 
-STATUS_BEACON_SIZE_BYTES = 24
-STATUS_BEACON_PACKET_LENGTH_INFO = b"\x0e\x00"
-STATUS_BEACON_MODULE_ID = b"\x00"
-STATUS_BEACON_PACKET_TYPE = b"\x00"
+STATUS_BEACON_SIZE_BYTES = 28
+HANDSHAKE_RESPONSE_SIZE_BYTES = 28
+DEFAULT_SIMULATOR_STATUS_CODE = bytes(4)
+TIMESTAMP_LENGTH_BYTES = 8
+CHECKSUM_LENGTH_BYTES = 4
 
 
 def append_checksum_to_data_packet(data_bytes: bytes) -> bytes:
-    return data_bytes + crc32(data_bytes).to_bytes(4, byteorder="little")
+    return data_bytes + crc32(data_bytes).to_bytes(
+        CHECKSUM_LENGTH_BYTES, byteorder="little"
+    )
+
+
+def create_data_packet(
+    timestamp: int,
+    module_id: int,
+    packet_type: int,
```

Comment:
should this just be a standalone (or static) function in `mc_simulator` (that gets called by `_send_data_packet`)?  it seems like there's a lot of duplicated code between this and the code in the simulator...so we're not really testing, we may just be copy/pasting code into the tests.  and then a specific single test that the function creates the expected bytes with just a single set of inputs?